frontend/src/components/pages/Scheduling/NextButton.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Button } from "@chakra-ui/react";
+import { NextButtonProps } from "./types";
+
+export default function NextButton({isBeingEdited, go, canSubmit, handleNext}: NextButtonProps) {
+  return (
+    <div style={{display: "flex", justifyContent: "flex-end"}}>
```

Comment:
nit: to adhere to ChakraUI
```suggestion
    <Flex justify="flex-end">
```